### ISSUE TITLE
Thumbnail MP3 error

### ISSUE THREAD

**Issue by XDjackieXD** [rg3/youtube-dl#5526]
When adding a thumbnail with --embed-thumbnail ffmpeg errors.
I'm running the latest git version (cloned 20 minutes ago)

This is the verbose output:
```
jakob@JakobUbuntu:~/Musik/SparxMusic$ ../youtube-dl/youtube-dl --audio-quality 0 -o "%(title)s.%(ext)s" -c -i -w --audio-format mp3 --embed-thumbnail --add-metadata https://soundcloud.com/sparxmusic/sparx-refraction-ft-adhesive --verbose
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--audio-quality', u'0', u'-o', u'%(title)s.%(ext)s', u'-c', u'-i', u'-w', u'--audio-format', u'mp3', u'--embed-thumbnail', u'--add-metadata', u'https://soundcloud.com/sparxmusic/sparx-refraction-ft-adhesive', u'--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.04.17
[debug] Python version 2.7.9 - Linux-3.19.0-15-generic-x86_64-with-Ubuntu-15.04-vivid
[debug] exe versions: avconv 11.2-6, avprobe 11.2-6, ffmpeg 2.5.4-1, ffprobe 2.5.4-1, rtmpdump 2.4
[debug] Proxy map: {}
[soundcloud] sparxmusic/sparx-refraction-ft-adhesive: Resolving id
[soundcloud] sparxmusic/sparx-refraction-ft-adhesive: Downloading info JSON
[soundcloud] 90044980: Downloading track url
[soundcloud] 90044980: Checking fallback video format URL
[soundcloud] 90044980: Checking http_mp3_128_url video format URL
[debug] Invoking downloader on u'https://ec-media.sndcdn.com/JcC4qYWxF6mN.128.mp3?f10880d39085a94a0418a7ef69b03d522cd6dfee9399eeb9a522039566f0ba3e0d0e94ab544bd8e8371cdd653d7445d143e4bba2f35f23c0637d06720812fb193420ee8852'
[download] Sparx - Refraction (Ft. Adhesive Wombat).mp3 has already been downloaded
[download] 100% of 3.72MiB
[ffmpeg] Adding metadata to 'Sparx - Refraction (Ft. Adhesive Wombat).mp3'
[debug] ffmpeg command line: avconv -y -i 'Sparx - Refraction (Ft. Adhesive Wombat).mp3' -c copy -metadata 'comment=Funky stuff & Sharped Basses in an 8-bit galaxy, this is Refraction ! 

We worked on it 35 Hours

SOUNDCLOUD : http://m1p.fr/wNa
YOUTUBE : m1p.fr/v6r
TWITTER : m1p.fr/v6s
FACEBOOK : m1p.fr/v6x
' -metadata 'description=Funky stuff & Sharped Basses in an 8-bit galaxy, this is Refraction ! 

We worked on it 35 Hours

SOUNDCLOUD : http://m1p.fr/wNa
YOUTUBE : m1p.fr/v6r
TWITTER : m1p.fr/v6s
FACEBOOK : m1p.fr/v6x
' -metadata artist=Sparx -metadata 'title=Sparx - Refraction (Ft. Adhesive Wombat)' -metadata date=20130429 -metadata purl=http://soundcloud.com/sparxmusic/sparx-refraction-ft-adhesive 'Sparx - Refraction (Ft. Adhesive Wombat).temp.mp3'
[ffmpeg] Adding thumbnail to "Sparx - Refraction (Ft. Adhesive Wombat).mp3"
[debug] ffmpeg command line: avconv -y -i 'Sparx - Refraction (Ft. Adhesive Wombat).mp3' -i 'Sparx - Refraction (Ft. Adhesive Wombat).thumb.mp3' -c copy -map 0 -map 1 -metadata:s:v 'title="Album cover"' -metadata:s:v 'comment="Cover (Front)"' 'Sparx - Refraction (Ft. Adhesive Wombat).temp.mp3'
ERROR: Sparx - Refraction (Ft. Adhesive Wombat).thumb.mp3: could not find codec parameters
Traceback (most recent call last):
  File "../youtube-dl/youtube-dl/youtube_dl/YoutubeDL.py", line 1520, in post_process
    files_to_delete, info = pp.run(info)
  File "../youtube-dl/youtube-dl/youtube_dl/postprocessor/embedthumbnail.py", line 44, in run
    self.run_ffmpeg(filename, temp_filename, options)
  File "../youtube-dl/youtube-dl/youtube_dl/postprocessor/ffmpeg.py", line 152, in run_ffmpeg
    self.run_ffmpeg_multiple_files([path], out_path, opts)
  File "../youtube-dl/youtube-dl/youtube_dl/postprocessor/ffmpeg.py", line 148, in run_ffmpeg_multiple_files
    raise FFmpegPostProcessorError(msg)
FFmpegPostProcessorError

jakob@JakobUbuntu:~/Musik/SparxMusic$ 
```

**Comment by XDjackieXD** [comment]
It works now! Thanks for the quick fix!

**Comment by dstftw** [comment]
This has been fixed in youtube-dl 2015.04.26. See our [FAQ](https://yt-dl.org/update) if you have problems updating.
